### PR TITLE
Add support for Qlogic NX2

### PR DESCRIPTION
**Pull request by martbhell**
Like this one: HP FlexFabric 10Gb 2-port 534FLR-SFP+ Ad

 - #CCCP-2288

### CODE REVIEW

File: tasks/qlogic_network_adapter.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- assert: { that: "ansible_system_vendor == 'HP'" }
```
Comment (oscarkraemer) [inline tasks/qlogic_network_adapter.yml:2]: I don't know the QLogic seems so weird, Dell seems also to be selling the same card. Maybe there is a better assert(s)?

lspci |grep 'NetXtreme II BCM57810'
05:00.0 Ethernet controller: Broadcom Limited NetXtreme II BCM57810 10 Gigabit Ethernet (rev 10)

**Review by oscarkraemer — COMMENTED** [review]
I don't have very strong opinions but I would prefer not to install the firmware if the node does not have the hardware.

File: tasks/qlogic_network_adapter.yml (hunk 1 above)
Comment (martbhell) [reply tasks/qlogic_network_adapter.yml:2]: For sure there are better asserts, good point. I'll improve that piece.

Comment (martbhell) [reply tasks/qlogic_network_adapter.yml:2]: Added some asserts

**Review by oscarkraemer — APPROVED** [review]
Ok looks fine

**Pull request merged**